Autor.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Autor {
+    private string $nombre;
+    private string $nacionalidad;
+
+    public function __construct(string $nombre, string $nacionalidad) {
+        $this->nombre = $nombre;
+        $this->nacionalidad = $nacionalidad;
+    }
+
+    // getters
+    public function getNombre(): string { return $this->nombre; }
```

Comment:
```suggestion
    public function getNombre(): string {
        return $this->nombre;
    }
```
Así es más legible, puede guiarse en el formato PSR-12